org.rdkit.knime.wizards/META-INF/MANIFEST.MF

```diff
@@ -17,7 +17,7 @@ Require-Bundle: org.eclipse.core.runtime;bundle-version="[3.20.0,4.0.0)",
  org.eclipse.ui.views.properties.tabbed;bundle-version="[3.9.0,4.0.0)",
  org.eclipse.gef;bundle-version="[3.11.0,4.0.0)",
  org.eclipse.pde;bundle-version="[3.13.0,4.0.0)",
- org.apache.commons.io;bundle-version="[2.8.0,3.0.0)"
+ org.apache.commons.io;bundle-version="[2.6.0,3.0.0)"
```

Comment:
It seems a bit weird to step the version back here. I assume there's a good reason for it?

Reply:
Yes, the reason is that the former version increase was wrong or let's say "too tight". I tend to forget that the Wizard has a different target Eclipse environment than the rest, because the Wizard is used in the IDE to develop KNIME nodes, so it does not fall under the same constraints as the nodes plugin. In fact, it has to be much more flexible in its dependencies to allow the developer to install it. And on my own IDE, I got problems reported when being on version 2.8.0...